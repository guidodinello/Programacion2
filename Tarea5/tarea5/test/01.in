# Implementar estaEnColCadenas, insertarEnColCadenas del mÃ³dulo colCadenas
estaEnColCadenas 0 1
insertarEnColCadenas (0,2.0) 1
estaEnColCadenas 0 0
estaEnColCadenas 0 1
estaEnColCadenas 0 2
estaEnColCadenas 1 1
estaEnColCadenas 2 1
insertarEnColCadenas (8,6.0) 0
insertarEnColCadenas (5,1.0) 1
insertarEnColCadenas (3,7.0) 9
estaEnColCadenas 0 1
estaEnColCadenas 3 1
estaEnColCadenas 5 1
estaEnColCadenas 0 0
estaEnColCadenas 8 0
estaEnColCadenas 0 9
estaEnColCadenas 3 9
insertarEnColCadenas (40,3.0) 5
insertarEnColCadenas (41,3.0) 5
insertarEnColCadenas (42,3.0) 5
insertarEnColCadenas (43,3.0) 5
insertarEnColCadenas (44,3.0) 5
insertarEnColCadenas (45,3.0) 5
insertarEnColCadenas (46,3.0) 5
insertarEnColCadenas (47,3.0) 5
insertarEnColCadenas (48,3.0) 5
insertarEnColCadenas (49,3.0) 5
insertarEnColCadenas (50,3.0) 5
insertarEnColCadenas (51,3.0) 5
estaEnColCadenas 40 5
estaEnColCadenas 51 5
Fin
