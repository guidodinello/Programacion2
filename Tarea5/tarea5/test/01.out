1>#  Implementar estaEnColCadenas, insertarEnColCadenas del módulo colCadenas.
2>0 no está en la cadena 1.
3>Se insertó 0 en la cadena 1.
4>0 no está en la cadena 0.
5>0 está en la cadena 1.
6>0 no está en la cadena 2.
7>1 no está en la cadena 1.
8>2 no está en la cadena 1.
9>Se insertó 8 en la cadena 0.
10>Se insertó 5 en la cadena 1.
11>Se insertó 3 en la cadena 9.
12>0 está en la cadena 1.
13>3 no está en la cadena 1.
14>5 está en la cadena 1.
15>0 no está en la cadena 0.
16>8 está en la cadena 0.
17>0 no está en la cadena 9.
18>3 está en la cadena 9.
19>Se insertó 40 en la cadena 5.
20>Se insertó 41 en la cadena 5.
21>Se insertó 42 en la cadena 5.
22>Se insertó 43 en la cadena 5.
23>Se insertó 44 en la cadena 5.
24>Se insertó 45 en la cadena 5.
25>Se insertó 46 en la cadena 5.
26>Se insertó 47 en la cadena 5.
27>Se insertó 48 en la cadena 5.
28>Se insertó 49 en la cadena 5.
29>Se insertó 50 en la cadena 5.
30>Se insertó 51 en la cadena 5.
31>40 está en la cadena 5.
32>51 está en la cadena 5.
33>Fin.
